### PR TITLE
Busola deployments update check

### PR DESCRIPTION
**Pull request by Sawthis**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: templates/data/busola-web-deployment-check-data.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+              preset-sa-vm-kyma-integration: "true"
+              preset-gc-project-env: "true"
+        jobConfigs:
+          - repoName: kyma-project/busola
+            jobs:
+              # presubmits
+              - jobConfig:
+                  name: "pre-busola-web-deployment-check"
+                  optional: false
+                  run_if_changed: "^core/|^core-ui/|^service-catalog-ui/|^components/|^shared/|^package.json"
```
Comment (parostatkiem) [inline templates/data/busola-web-deployment-check-data.yaml:34]: We don't have the `components` folder anymore

**Review by parostatkiem — CHANGES_REQUESTED** [review]

**Review by parostatkiem — DISMISSED** [review]

**Review by dekiel — CHANGES_REQUESTED** [review]

File: templates/data/busola-backend-deployment-check-data.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          extra_refs_kyma-local:
+            extra_refs:
+              local-kyma:
+                - org: "kyma-incubator"
+                  repo: "local-kyma"
+                  path_alias: "github.com/kyma-incubator/local-kyma"
+                  base_ref: "main"
+          jobConfig_default:
+            path_alias: github.com/kyma-project/busola
+          jobConfig_periodic:
```
[the remark this answers is not in the document]
Comment (Sawthis) [reply templates/data/busola-backend-deployment-check-data.yaml]: fixed

File: templates/data/busola-web-deployment-check-data.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+          extra_refs_kyma-local:
+            extra_refs:
+              local-kyma:
+                - org: "kyma-incubator"
+                  repo: "local-kyma"
+                  path_alias: "github.com/kyma-incubator/local-kyma"
+                  base_ref: "main"
+          jobConfig_default:
+            path_alias: github.com/kyma-project/busola
+          jobConfig_periodic:
```
[the remark this answers is not in the document]
Comment (Sawthis) [reply templates/data/busola-web-deployment-check-data.yaml]: fixed

**Review by dekiel — APPROVED** [review]

**Pull request merged**